src/ray/gcs/tables.h

```diff
@@ -422,20 +426,10 @@ class Set : private Log<ID, Data>,
   Status Remove(const JobID &job_id, const ID &id, std::shared_ptr<DataT> &data,
                 const WriteCallback &done);
 
-  Status Lookup(const JobID &job_id, const ID &id, const Callback &lookup) {
-    return Log<ID, Data>::Lookup(job_id, id, lookup);
-  }
-
-  void Delete(const JobID &job_id, const ID &id) { Log<ID, Data>::Delete(job_id, id); }
-
-  void Delete(const JobID &job_id, const std::vector<ID> &ids) {
-    Log<ID, Data>::Delete(job_id, ids);
-  }
-
   Status Subscribe(const JobID &job_id, const ClientID &client_id,
                    const NotificationCallback &subscribe,
                    const SubscriptionCallback &done) {
-    return Log<ID, Data>::SubscribeWithNotificationMode(job_id, client_id, subscribe,
+    return Log<ID, Data>::Subscribe(job_id, client_id, subscribe,
```

Comment:
I guess we could also do `using Log<...>::Subscribe` here.

Reply:
For `Set`, I only want to expose the overloaded `Subscribe` function with the `NotificationCallback` parameter. If we use `using Log<...>::Subscribe` here, both overloaded functions are exposed.